### ISSUE TITLE
Synthetic Challenges in Miyaura Borylation of Bromo-Thienopyrimidine

### ISSUE THREAD

**Issue by yinuowang0812** [OpenSourceMalaria/Series3#36]
We need some help or suggestion about Miyaura borylation reaction.

We synthesised a series of final compounds via Suzuki coupling with Br-aminothienopyrimidine (our core 4) and B(pin)-aryl sulfonamides in OSM3 analogues synthesis (detailed protocol can be found in [Yinuo's ELN YNW5](https://uk-mynotebook.labarchives.com/share/Yinuo%2520Wang/MTE0LjR8MTU5NjIvODgvVHJlZU5vZGUvMTgzNDU4MTJ8MjkwLjQ=)). But now we have a little roadblock - if we can borylate the aminothienopyrimidine core (Br to boronate) then we could directly couple a bunch of quite expensive building blocks. 

<img width="888" alt="Screenshot 2024-03-04 at 16 25 39" src="https://github.com/OpenSourceMalaria/Series3/assets/102600515/de70acc4-7f28-4bd1-a5bc-e032fc889b24">

So here is what I've done so far: 

Note: The purity of Compound 4 and B(Pin)2 has been checked and they were used as controls in LCMS and NMR analyses.

1. Direct borylation: ([ELN YNW63](https://uk-mynotebook.labarchives.com/share/Yinuo%2520Wang/MjIzLjZ8MTU5NjIvMTcyL1RyZWVOb2RlLzI3NjY5MTc2NXw1NjcuNg==)):

I've done with various conditions for the direct Miyaura borylation of compounds 4 and B(Pin)2 based on the paper ([Thienophene borylation](https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01805)) and ([Amino included borylation](https://doi.org/10.1021/jm800318d)). However, the LCMS results under all the borylation conditions tested below showed m/z = 152 (dehalogenation, a known [side reaction](https://pubs.rsc.org/en/content/articlehtml/2013/cc/c3cc46271d) with [Pd catalysts](https://pubs.acs.org/doi/full/10.1021/acs.joc.9b01350)) and m/z = 255 (suggesting unreacted B(pin)2).

<img width="769" alt="Screenshot 2024-03-04 at 16 24 33" src="https://github.com/OpenSourceMalaria/Series3/assets/102600515/c0968464-eb1b-4159-bb73-cda1866f9fbf">

Conditions tested:

(1) Pd(PPH3)4, potassium acetate, diethyl ether/H2O, 70 °C, overnight. ([patent](https://worldwide.espacenet.com/patent/search/family/059790064/publication/CN107163022A?q=pn%3DCN%20107163022))
(2) Pd(dppf)Cl2·DCM, potassium acetate, dioxane, 70 °C /100 °C, overnight. ([DMK180-1](https://au-mynotebook.labarchives.com/share/Dana%2520Klug/NDIyLjV8Njk2ODYvMzI1L1RyZWVOb2RlLzQwMjQ0MTU5MDB8MTA3Mi41))
(3) Pd(dppf)Cl2·DCM, potassium acetate, toluene, 80 °C, 24 hrs. ([lit](https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01805))

2. Boc-protected borylation: ([ELN YNW66](https://uk-mynotebook.labarchives.com/share/Yinuo%2520Wang/MjMxLjR8MTU5NjIvMTc4L1RyZWVOb2RlLzMwNzA5NzIxMXw1ODcuNA==))

I then attempted to protect the amine at the 4-position with a Boc group before proceeding with the borylation.  But the results indicated both Boc deprotection and dehalogenation occurred.  LCMS also found m/z = 152 (dehalogenation) and m/z = 255 (B(pin)2 remained).   Two of my colleagues have observed that Pd coupling can lead to the deprotection of NH-Boc groups.

<img width="726" alt="Screenshot 2024-03-04 at 16 12 25" src="https://github.com/OpenSourceMalaria/Series3/assets/102600515/7c0e9e1e-7135-4f6c-8069-8d1cefa0d8ae">

3. Selective borylation: ([ELN YNW68](https://uk-mynotebook.labarchives.com/share/Yinuo%2520Wang/MjQwLjV8MTU5NjIvMTg1L1RyZWVOb2RlLzIwNDM5Nzg3Mzd8NjEwLjU=))

Referencing patents ([1](https://worldwide.espacenet.com/patent/search/family/083405038/publication/CN115141209A?q=CN115141209A)) and ([2](https://worldwide.espacenet.com/patent/search/family/061692082/publication/WO2018160878A1?q=pn%3DWO2018160878A1)), the borylation showed more selectivity to-Br then -Cl.  So I attempted to borylate the intermediate, 6-bromo-4-chlorothienopyrimidine, but no desired product was detected in LCMS. LCMS found minor trace of m/z = 170 (dehalogenation) and m/z = 255 (B(pin)2 remained). 

<img width="772" alt="Screenshot 2024-03-04 at 16 15 39" src="https://github.com/OpenSourceMalaria/Series3/assets/102600515/b77910e2-f581-47ab-becc-7ff7f6a5f3b4">

In summary, all attempted borylation conditions resulted in dehalogenation. It's unclear why Miyaura borylation is ineffective on aminothienopyrimidine, yet it succeeds on thienophene and amino-aryl rings separately (both using Pd and similar temperatures). Feel free to comment if you have any idea or advice :)

<img width="899" alt="Screenshot 2024-03-04 at 16 28 37" src="https://github.com/OpenSourceMalaria/Series3/assets/102600515/f2a45793-7618-4554-824e-ace588c0d27a">

**Comment by yinuowang0812** [comment]
Hi @davidc54 , thank you for your comment! Very helpful. 

Do you think the protodeboronation happened after the transmetalation, and then pinacolborane gets protonolysis by H2O or something else? I wondered if the dehalogenation could result from β-elimination of acetate.

The Ir catalyst paper is very interesting. It includes examples of borylating 2-substituted pyrimidines. I will add it into my synthetic plan for sure. 

Before we look into other ways of doing things, we really want to sort out this Miyaura borylation reaction. I'll dig into what's been tried before and we'd like to crack this reaction first.

**Comment by paulkingau** [comment]
I did a database search for this reaction (I searched the core structure plus boron). There are only two compounds in the entire literature with boron in that position and are both Chinese patents. You may wish to look at these patents: CN 115716840 A and CN 113214326 A.

CN 115716840 A
![image](https://github.com/OpenSourceMalaria/Series3/assets/162650039/355dab61-260a-4a01-9172-7e931e6be8bb)

There does appear to be a mistake in the drawing here.

Google Patents translation: "INT 6-1 Synthesis: n is a radical of hydrogen 2 INT 3 (1.00g, 2.01mmol), pinacolborane (5.14g, 40.20mmol), triethylamine (0.61g, 6.03mmol), 2-dicyclohexylphosphine-2 ',4',6' -triisopropylbiphenyl (0.19g, 0.40mmol) and tris (dibenzylideneacetone) dipalladium (0.18g, 0.20mmol) were added to dioxane (16.0 ml) under protection, and stirring was carried out at 105 ℃ for 0.5 hour, and LC-MS monitored for the absence of a raw material. The reaction solution was used in the next step without purification."

CN 113214326 A
![image](https://github.com/OpenSourceMalaria/Series3/assets/162650039/36b2e5ca-e1f8-4980-9a9e-a871f75bc85f)

**Comment by davidc54** [comment]
[paulkingau](https://github.com/paulkingau) Great idea, its never a good sign when you see this small a number of prior examples in a SciFinder search! To me, the examples shown look more achievable anyway due to the absence of a free amino group on the pyrimidine (to my knowledge, no one has made clear "rules" for which of these compound should be stable and which should not, would be delighted to be pointed to a publication that says otherwise).

[yinuowang0812](https://github.com/yinuowang0812) Yes I would expect that the reaction proceeds to the desired boronate (inspired by your thiophene literature result above), but under the Miyaura conditions the product is so unstable that it is destroyed before you can work it up. Dehalogenation is also possible as you suggest, perhaps you could test this by running the reaction in the absence of B2Pin2, if you see the same amount of the C-H product then you are correct :)

Protodeboronation is an interesting reaction thats been studied a fair bit due to it being the cause of problems such as this, a couple of starter refs can be found in the work of Guy Lloyd-Jones et al.:
https://pubs.acs.org/doi/10.1021/jacs.6b03283 (take a look at the thiazole example in the graphical abstract)
https://pubs.acs.org/doi/10.1021/jacs.7b07444

Other ideas for this reaction: have you tried other sources of boron, such as bis-boric acid, B2Cat2, B2Epin2 (ethyl version of pinacol)
Have you tried additives/co-catalysts for this reaction? Other solvents? Perhaps you could speed up the reaction to make it go at a lower temperature.

Have you tried deprotonative borylation using BuLi and a boronate ester such as isopropoxyBPin? (https://www.sigmaaldrich.com/GB/en/product/aldrich/417149)

**Comment by mlc-syg** [comment]
At the CRO I work for we've found the conditions in this paper with XPhos precatalysts to be superior in the majority of cases often overcoming issues with dehalogenation:
https://pubs.acs.org/doi/pdf/10.1021/acs.joc.0c01758

Alternatively, you could consider the 2-step 1 pot borylation/Suzuki method with your sulfonamide as the reagent that is borylated...

**Comment by jhjensen2** [comment]
Perhaps [this paper ](doi.org/10.1021/ja1073799) could help?

**Comment by mattodd** [comment]
Hi @paulkingau !

**Comment by yinuowang0812** [comment]
Hi @jhjensen2 the link you shared  is unfortunately 404

**Comment by jhjensen2** [comment]
Sorry, should work now

**Comment by yinuowang0812** [comment]
Hi guys, many thanks for sharing your experiences and providing diverse suggestions for my borylation issue. I’ve summarized the ideas from all platforms and compiled them here to keep everyone on the same page. Please don’t hesitate to share if you have more ideas!

1. Not Isolating the Borylated Intermediate:
    -  [This paper](https://pubs.acs.org/doi/full/10.1021/jo301642v#) has been repeatedly mentioned with the idea of a “Two-Step, One-Pot Palladium-Catalyzed Borylation,” which involves carrying out the Suzuki coupling after borylation in one pot without isolating the boronate intermediate. It describes a direct Borylation + Suzuki coupling reaction to yield the final compounds. Both conditions below are worth trying.
    - The free amine from the theinopyrimidine (our core) could be protected by DMF·DMA, which can be removed by mild acid at the end.
![Screenshot 2024-03-12 at 14 34 18](https://github.com/OpenSourceMalaria/Series3/assets/102600515/a1c2383c-ff8f-4462-b544-24d0d211268b)

2. Optimising the Reagents (can also be combined with the first optimisation):
    - The stability of boronate needs to be considered.  Can try altering the boronate source such as B2(OH)2, B2(EPin)2, B2Cat2, (BF3K salts along with)BMIDA…
    - Optimisation of the Base (https://pubs.acs.org/doi/10.1021/acs.joc.0c01758?ref=pdf) such as  2-ethyl hexanoate.

3. Optimise the method:
    - Use BuLi deprotonated and use Triisopropyl borate
    - Stille coupling / Stannylation (https://chemistry-europe.onlinelibrary.wiley.com/doi/full/10.1002/ejoc.201701463)
    - Ir-catalysed C-H borylation (https://onlinelibrary.wiley.com/doi/full/10.1002/anie.202001520)
    - Ni-catalysed reductive C-B coupling of bromoborane (https://pubs.acs.org/doi/10.1021/jacs.4c01450?ref=pdf)


The lack of literature or patents on the borylation of aminopyrimidine/thienophene combo poses a challenge for this reaction. But I believe the posts here already provide a broad perspective on borylation conditions, and I will optimise the reaction next following these ideas. I'll keep you guys updated :)

**Comment by Jamesw350** [comment]
If the goal is the aryl-aryl product and med chem scale, fighting with a challenging boronation/unstable boronate isn’t a good use of time, a process chemist can solve that problem later when you have identified the clinical candidate.  Stanylation seems like a suitable alternative. Li-Br exchange on either partner then charge Bu3Sn-Cl to hopefully give you a stable coupling partner. Initial deprotonation of the NH2 might be needed or 2,6-hexanedione can be used as a protecting group to hide the acidic protons. The hexanedione protecting group could also be a viable option to avoid formation of the protonated impurity in your borylations.
